clang/docs/ReleaseNotes.rst

```diff
@@ -249,6 +249,7 @@ DWARF Support in Clang
 
 Floating Point Support in Clang
 -------------------------------
+- Add ``__builtin_elementwise_length``builtin for floating point types only.
```

Comment:
Same comment this doesn't match `__builtin_hlsl_elementwise_length`. Also These release notes are probably not intended for the HLSL dialect of clang.  Should we remove this line?

Reply:
Ah, this was a habit that seems to no longer be justified. I've removed changes to both LangaugeExtensions and ReleaseNotes